app/site/metrics.js

```diff
@@ -107,6 +107,54 @@ export const communityMetrics = {
     format: Formatter.percentWithNoDecimal,
     fullFormat: Formatter.percentWithOneDecimal,
   },
+  rent_burdened_2015: {
+    name: "2015 Rate of Rent Burdened Households",
+    units: "of households",
+    format: Formatter.percentWithOneDecimal(),
```

Comment:
Delete the parentheses from all of the format and fullFormat lines.

The difference is:

- If you include the parentheses, that calls the method right now
- If you leave out the parentheses, that gives a variable for the method

We want the latter, that way we can pass the format function to whatever code will actually call it and apply the formatting.

Reply:
Deleted!